myproject/supermarket_manager/views.py

```diff
@@ -485,6 +485,8 @@ def update_account(request, account_id):
         data = json.loads(request.body)
 
         if request.method == "PUT":
+            # Update account information but do not change password
```

Comment:
Should comment above the `if` statement.

Reply:
Done